### DISCUSSION TITLE
Any officially supported VSCode extension available?

### DISCUSSION THREAD

**Discussion by mrdrivingduck** [community/community#72929] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

When I browse the code on GitHub web page, the UI of code search and the result is much better for me than the searching page of VSCode. How soon will a VSC extension be available?

**Comment by look** [comment, marked as answer]
Glad you like the code search UI on GitHub! We've heard from a few people that they'd like to use GitHub code search inside VS Code (and other editors), but we haven't put this on our roadmap yet. We plan to release a better API that would enable ourselves or others to build a nice extension, however work on the API has taken a back seat to other necessary work. We still plan to build it, though.

**Reply by mrdrivingduck** [reply]
Glad to know, thanks.
